website/docs/plugins/ray_launcher.md

```diff
@@ -3,108 +3,51 @@ id: ray_launcher
 
-This plugin requires Hydra 1.0 to install:
-```commandline
-$ pip install hydra-ray-launcher --pre
-```
 
-After installation, override the Hydra launcher via your command line to activate one of the launchers:
+### Installation
 
 ```commandline
-# AWS Ray Launcher
-$ python my_app.py hydra/launcher=ray_aws
-
-# Local Ray Launcher
-$ python my_app.py hydra/launcher=ray_local
+$ pip install hydra-ray-launcher --pre
 ```
 
-#### ray_aws launcher
+### Usage
+Once installed, add `hydra/launcher=ray_aws` or `hydra/launcher=ray_local` to your command line. Alternatively, override `hydra/launcher` in your config:
 
-`ray_aws` launcher is built on top of ray's [autoscaler cli](https://docs.ray.io/en/latest/autoscaling.html). To get started, you need to 
-config your AWS credentials first, tutorials can be found [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
-You can run an initial check on credential configuration running the following command.
-```commandline
- python -c 'import boto3;boto3.client("ec2")'
+```yaml
+defaults:
+  - hydra/launcher: ray_aws
 ```
-:::caution
-`ray autoscaler` expects your AWS user has certain permissions for `EC2` and `IAM`. This [issue](https://github.com/ray-project/ray/issues/9327) provides some contexts.
-:::
 
 
-`ray autoscaler` expects a yaml file to provide specs for the new cluster, which we've schematized in `hydra_ray_launcher.hydra_plugins.hydra_ray_launcher.conf.__init__.RayClusterConf`, 
-The plugin defaults are in `conf/hydra/launcher/ray_aws.yaml`. You can override the default values in your app config or from command line.
+### `ray_aws` launcher
+
+`ray_aws` launcher is built on top of ray's [autoscaler cli](https://docs.ray.io/en/latest/autoscaling.html). To get started, you need to 
+[config your AWS credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
 
 :::caution
-This plugins depends on `cloudpickle`, as a result please make sure your local machine and the AWS cluster runs the same version of: `ray`, `hydra-core` and `python`. You can install the desired software version by overriding `RayClusterConf.setup_commands`.
+`ray autoscaler` expects your AWS credentials has certain permissions for [`EC2`](https://aws.amazon.com/ec2) and [`IAM`](https://aws.amazon.com/iam). This [ray issue](https://github.com/ray-project/ray/issues/9327) provides some contexts.
 :::
 
-Now we can go ahead and run `train.py` using `ray_aws` launcher
+`ray autoscaler` expects a yaml file to provide configuration for the EC2 cluster; we've schematized the configs in [`hydra_ray_launcher.hydra_plugins.hydra_ray_launcher.conf.__init__.RayClusterConf`](https://github.com/facebookresearch/hydra/blob/master/plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py), 
```

Comment:
`__init__` is not a part of the module to the conf.
(try to import that thing, it will not work).

Just give the class name with a link.